File: Practicing/Java NetBeans/Matrices/src/main/java/clases/MultiplicacionMatrices.java
package clases;

public class MultiplicacionMatrices {

    public static void main(String[] args) {

        int matriz_uno[][] = new int[2][3];
        int matriz_dos[][] = new int[3][2];
        int matriz_tres[][] = new int[2][2];

        //Llenado de la matriz uno
        for (int i = 0; i < 2; i++) {
            for (int j = 0; j < 3; j++) {
                matriz_uno[i][j] = (int) (Math.random() * 5);
            }
        }

        //Llenado de la matriz dos
        for (int i = 0; i < 3; i++) {
            for (int j = 0; j < 2; j++) {
                matriz_dos[i][j] = (int) (Math.random() * 5);
            }
        }

        //Multiplicacion de matrices
        for (int i = 0; i < 4; i++) {

            if (matriz_tres[0][0] == 0) {

                int temporal = (matriz_uno[0][0] * matriz_dos[0][0]) + (matriz_uno[0][1] * matriz_dos[1][0])
                        + (matriz_uno[0][2] * matriz_dos[2][0]);

                matriz_tres[0][0] = temporal;

            } else if (matriz_tres[1][0] == 0) {

                int temporal = (matriz_uno[1][0] * matriz_dos[0][0]) + (matriz_uno[1][1] * matriz_dos[1][0])
                        + (matriz_uno[1][2] * matriz_dos[2][0]);

                matriz_tres[1][0] = temporal;

            } else if (matriz_tres[0][1] == 0) {

                int temporal = (matriz_uno[0][0] * matriz_dos[0][1])
                        + (matriz_uno[0][1] * matriz_dos[1][1])
                        + (matriz_uno[0][2] * matriz_dos[2][1]);

                matriz_tres[0][1] = temporal;

            } else if (matriz_tres[1][1] == 0) {

                int temporal = (matriz_uno[1][0] * matriz_dos[0][1])
                        + (matriz_uno[1][1] * matriz_dos[1][1])
                        + (matriz_uno[1][2] * matriz_dos[2][1]);

                matriz_tres[1][1] = temporal;
            }
        }

        //Impresion de la matriz
        for (int i = 0; i < 3; i++) {

            if (i < 2) {

                for (int j = 0; j < 3; j++) {

                    System.out.print("[ " + matriz_uno[i][j] + " ]");

                }
                if (i == 1) {

                    System.out.print("   x   ");

                } else {

                    System.out.print("       ");

                }

            } else {

                System.out.print("                      ");

            }

            for (int j = 0; j < 2; j++) {

                System.out.print("[ " + matriz_dos[i][j] + " ]");

            }

            if (i == 1) {

                System.out.print("   =   ");

            } else {

                System.out.print("       ");

            }
            
            if(i < 2){
             
                for (int j = 0; j < 2; j++) {
                    
                    System.out.print("[ " +  matriz_tres[i][j] + " ]");
                    
                }
                
            }
            
            System.out.println("");

        }

    }

}

File: Practicing/Python/MultiplicacionMatrices.py
filas_Uno, filas_Dos, columnas_Uno, columnas_Dos = 0, 0, 0, 0
matriz_Uno, matriz_Dos = [], []

print("")
filas_Uno = int(input("Filas de la matriz A: "))
columnas_Uno = int(input("Columnas de la matriz A: "))
print("")
filas_Dos = int(input("Filas de la matriz B: "))
columnas_Dos = int(input("Columnas de la matriz B: "))
print("")
print("")

if columnas_Uno == filas_Dos:

    #Llenado de la matriz uno
    print("Matriz A: ")
    for i in range(filas_Uno):
        aux = []
        for j in range(columnas_Uno):
            aux.append(int(input(f"Valor de la fila {i+1} columna {j+1}: ")))
        matriz_Uno.append(aux)
    print("")

    #Llenado de la matriz dos
    print("Matrtiz B: ")
    for i in range(filas_Dos):
        aux = []
        for j in range(columnas_Dos):
            aux.append(int(input(f"Valor de la fila {i+1} columna {j+1}: ")))
        matriz_Dos.append(aux)
    print("")

    #Impresion de las matrices
    #A
    print(" ** MATRIZ A **")
    for i in range(filas_Uno):
        for j in range(columnas_Uno):
            print(f"[{matriz_Uno[i][j]}]", end="")
        print("")
    print("")
    
    #B
    print(" ** MATRIZ B **")
    for i in range(filas_Dos):
        for j in range(columnas_Dos):
            print(f"[{matriz_Dos[i][j]}]", end="")
        print("")
    print("")

    #Calculo para las multiplicaciones para obtener la nueva matriz
    matriz_Tres = []
    for i in matriz_Uno:
        fila_aux = i;
        aux = []
        for j in range(columnas_Dos):
            for z in range(filas_Dos):
                

    print(f"{filas_Uno}x{columnas_Dos}")
else:
    print("Matrices incompatibles")